lib/BudgetButton/SecondPage.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class PageForName extends StatelessWidget {
```

Comment:
If these three are for the seperate pages the user moves through to set the budget name, date, and total amount. Maybe they can go in different files for readability?

Reply:
Just my thoughts though

Reply:
Yeah, I thought about that just for purely readability purposes but I forgot to do that at the end since it occurred to me after I pushed the changes.  If everyone wants me to do I have no problem changing that into separate files.

Reply:
If I can make a suggestion, could you put them all in one page and make the date default to the current day?

Reply:
Yup, I'm going to search it up but yeah I can probably make that the date defaults to current day. However, not everyone makes purchases in one day though so I just made it for the user to enter whatever date they spent it on, that's how I thought about it.

Reply:
Yea definitely, I just thought it might be a better experience if it shows the current date and the user could always change it.  Just a suggestion though, we could probably talk about it as a team.

Reply:
Yeah, sure.